docs/essentials/workspace.md

```diff
@@ -9,16 +9,16 @@ Create Workspaces for libraries, applications, services, teams or individuals. M
 
 ## Focus on Components
 
-Bit helps you focus on building components, as such the workspace has a minimal footprint on the codebase. There are only two files Bit needs to manage components and the workspace.
+Bit helps you focus on building components, and as such the workspace has a minimal footprint on the codebase. There are only two files Bit needs to manage components and the workspace.
 
-- `.bitmap` - Maps the location of Bit Components in the workspace.
-- `workspace.jsonc` - Configure dev-workflow and components.
+- `.bitmap` - Mapping of the location of Bit Components in the workspace.
+- `workspace.jsonc` - Configurations for dev-workflow and components.
 
 ## Flexible Directory Structure
 
-Bit Workspace is a monorepo that support nesting Bit Components in sub-directories. You can build a very-nested structure for your workspace, a flat structure, or decide where in your project you want deep nesting and where you don't.
+A it Workspace is a monorepo that supports nesting Bit Components in sub-directories. You can build a highly-nested structure for your workspace, a flat structure, or decide where in your project you want deep nesting and where you don't.
```

Comment:
@debs-obrien , should be `A Bit workspace`

Reply:
ok will fix. thanks